app/views/candidate_interface/degrees/base/_form.html.erb

```diff
@@ -0,0 +1,18 @@
+<%= f.govuk_text_field :subject, label: { text: t('application_form.degree.subject.label'), size: 'm' } %>
+<%= f.govuk_text_field :institution_name, label: { text: t('application_form.degree.institution_name.label'), size: 'm' } %>
+<%= f.govuk_radio_buttons_fieldset :grade, legend: { text: t('application_form.degree.grade.label'), tag: 'span' } do %>
+  <%= f.govuk_radio_button :grade, 'first', label: { text: t('application_form.degree.grade.first.label') } %>
+  <%= f.govuk_radio_button :grade, 'upper_second', label: { text: t('application_form.degree.grade.upper_second.label') } %>
+  <%= f.govuk_radio_button :grade, 'lower_second', label: { text: t('application_form.degree.grade.lower_second.label') } %>
+  <%= f.govuk_radio_button :grade, 'third', label: { text: t('application_form.degree.grade.third.label') } %>
+  <%= f.govuk_radio_button :grade, 'other', label: { text: t('application_form.degree.grade.other.label') } do %>
+    <%= f.govuk_text_field :other_grade, label: { text: t('application_form.degree.grade.other.conditional.label') }, width: 10 %>
+  <% end %>
```

Comment:
maybe we should create a helper or something containing the grades
`first`, `upper_second`, .. instead of hardcoding these in the view.
(have a look to `select_gcse_qualification_type_options` )

Reply:
Ooooo I like this. Will refactor!

Reply:
Actually it's kind of awkward in this case as we have conditional radio buttons with text fields...

Reply:
I'm going to merge this for now but if you do, I'll include it in my next PR. 👍